src/db_impl.cc

```diff
@@ -39,6 +40,12 @@ class TitanDBImpl::FileManager : public BlobFileManager {
       std::unique_ptr<WritableFile> f;
       s = db_->env_->NewWritableFile(name, &f, db_->env_options_);
       if (!s.ok()) return s;
+
+      // Set the io priority for WritableFile in Flush or GC.
+      // Or the ratelimiter will use the default priority IO_ALL
+      // which won't be limited in ratelimiter.
```

Comment:
Instead of here, I think it's better to explain the necessity to pass in IOPriority at `NewFile` declaration in src/blob_file_manager.h